README.md

```diff
@@ -16,32 +16,51 @@ environment:
 configmap:
   VAR_A: VALUE_A
   VAR_B: VALUE_B
+secrets: 
+  ENVIRONMENT_VAR:
+      secretRef: some-secret-reference
+      key: connectionString
 ```
 
+### secrets
+Secret objects in this configuration consist of name secretRef and key. These need to be put into the secrets array in the following manor
+```yaml
```

Comment:
manor = 'manner'?

Also might want to make it clear that this doesn't create secrets, but rather just references existing secrets and puts them into environment variables